apps/i18n/common/en_us.json

```diff
@@ -79,7 +79,7 @@
   "censusRequiredEmail": "Required. Please enter your email.",
   "censusRequiredSchool": "Required. Please enter your school information.",
   "censusRequiredSelect": "Required. Please select an option.",
-  "censusTextBased": "Text-based programming in a language such as Java, Python, C++, etc. (Excluding HTML or CSS)",
+  "censusTextBased": "Text-based programming in a language such as Java, Javascript, Python, C++, etc. (Excluding HTML or CSS)",
```

Comment:
I thought we always use JavaScript (with uppercase S) on our site?

Reply:
Oh yeah... guess we do. Sorry I didn't catch.